### ISSUE TITLE
Login on the portal requires to click login twice

### ISSUE THREAD

**Issue by prudhomm** [MSO4SC/MSOPortal#88]
When login on the portal, I need to click `Sign In ` twice on IDM service. 

This is not good for the  User Experience

**Comment by emepetres** [comment]
I understand it only happens on production (not in canary) and for @gdolle is working well.

@prudhomm I propose three actions to gather more information:
- could you try with @gdolle account on your computer to see what is happening?
- could you try accessing from home (different computer, different network)?
- could you temporarily change your password and send me the new one by email, so I can test with your user?

**Comment by emepetres** [comment]
Please note that httpS://portal.mso4sc.eu is IDM and http://portal.mso4sc.eu is Portal.

**Comment by VikingScientist** [comment]
<img width="778" alt="screenshot 2018-08-29 08 28 41" src="https://user-images.githubusercontent.com/3500376/44772252-a6060f00-ab6d-11e8-909b-fad204c6581a.png">

Sometimes you end up in this fine state where you're logged in and not logged in simultaneously.

**Comment by emepetres** [comment]
Yes @VikingScientist , I recently noted this behaviour as well. Could you please create a new issue for it? As far as possible, indicate how the problem can be reproduced, thanks.
